tritium-metrics/build.gradle

```diff
@@ -23,13 +23,14 @@ dependencies {
 
     testImplementation project(':tritium-test')
```

Comment:
See if we can share a single `TestTaggedMetricRegistries`

```suggestion
    testImplementation project(':tritium-registry')
    testImplementation project(':tritium-test')
```

Reply:
skipping as this ends up being nasty with gradle test dependencies, just going to leave the duplication for now